aead/CHANGELOG.md

```diff
@@ -5,6 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Fixed
+- minor documentation error in `AeadCore::TagSize` ([#1351])
+- fixup `hybrid-array` migration ([#1531])
+
+### Changed
+- Migrate to `doc_auto_cfg` ([#1370])
+- Exclude pre-1.60 crates from workspace ([#1380])
+- bump `crypto-common` to v0.2.0-pre; MSRV 1.65 ([#1384])
+- Bump `heapless` dependency to v0.8 ([#1398])
+- Bump `hybrid-array` to v0.2.0-pre.6 ([#1432])
+- Bump `crypto-common` to v0.2.0-pre.1 ([#1433])
+- Bump `crypto-common` to v0.2.0-pre.2 ([#1436])
+- Bump `hybrid-array` to v0.2.0-pre.8 ([#1438])
+- Bump `crypto-common` and `hybrid-array` ([#1469])
+- Bump `crypto-common` to v0.2.0-pre.5 ([#1496])
+
+### Added
+- enable `missing_debug_implementations` lint and add `Debug` impls ([#1411])
```

Comment:
We usually don't compile these for prereleases since the contents can change a lot prior to a release, but this is fine for now

Reply:
that was mostly to prepare for the stable release ^^